contracts/presets/ERC1155PresetMinterPauser.sol

```diff
@@ -53,7 +53,7 @@ contract ERC1155PresetMinterPauser is Context, AccessControl, ERC1155Burnable, E
     }
 
     /**
-     * @dev Batched variant of {mint}.
+     * @dev xref:ROOT:erc1155.adoc#batch-operations[Batched] variant of {mint}.
```

Comment:
I would make this more visible adding the phrase "See [Batched Operations](#)." at the end.

Reply:
I went with that one at first but didn't like how it looked in the docs, which is why I changed it for the current one. Happy to do either though.